keyboards/atreus/keymaps/yttyx/README.md

```diff
@@ -0,0 +1,81 @@
+# Overview
+
+A Balance 12 layout for the Atreus keyboard.
+
+Balance 12 was created by Sasha Viminitz. Please see [this page](https://mathematicalmulticore.wordpress.com/the-keyboard-layout-project/)
+for some background on the design of the layout.
+
+* The variant used here is a mirror of the original for left-handers
+* The central column of punctuation keys has been moved elsewhere
+* Home positions for the left and right forefingers are *T* and *A* respectively
+
+## To build
+
+> make atreus:yttyx
+
+## To flash
+
+> avrdude -p atmega32u4 -c avr109 -U flash:w:atreus_yttyx.hex -P /dev/ttyACM0
```

Comment:
`make atreus:yttyx:avrdude` should take care of this automatically (include port detection, IIRC)